repository.eselect.5.in

```diff
@@ -0,0 +1,169 @@
+.\" -*- coding: utf-8 -*-
+.\" Copyright 2019 Gentoo Authors
```

Comment:
Put your own name here, 'Gentoo Authors' is not correct for entirely external projects.